src/main.ts

```diff
@@ -67,13 +69,18 @@ async function run() {
       if (!repository) {
         core.setFailed("❌ Expecting repository metadata.")
         return;
-      }
+      } else
```

Comment:
This is not a very big deal, but is really not necessary, and including it would lead to a big amount of indentation down the body. Unrelated to the PR topic, too, I'm afraid...

Reply:
yeah spotted and resolved this one already, I believe. :)